packages/dev/core/src/assetContainer.ts

```diff
@@ -39,6 +39,26 @@ export class InstantiatedEntries {
      * List of new animation groups
      */
     public animationGroups: AnimationGroup[] = [];
+
+     /**
+     * Disposes the instantiated entries from the scene
+     */
+    public removeFromScene() {

```

Comment:
Actually, it would be more sensible to call it "dispose", as the `removeFromScene` function in `AssetContainer` is just removing the objects from the scene, it does not dispose them: that's what `AssetContainer.dispose` is doing. It would be weird to have `InstantiatedEntries.removeFromScene` disposing entries, whereas `AssetContainer.removeFromScene` only removes the objects from the scene.

@RaananW Why calling it `dispose` would be bad (as per your answer in the forum thread)?

Reply:
the only reason is semantic to be honest. Those objects will be disposed when the scene/engine is disposed. Also, you are not disposing the class itself, you are disposing the assets from the scene.

Just semantics really :-)

I am fine with both to be honest. Maybe we shouldn't dispose in general but use scene.remove????, similar to the way assetsContainer is doing it?

Reply:
Seems like scene.removeMesh / remove functions in general just remove the objects from the scene object and clears a parent if any 🤔 

But is that enough for them to be GC'd after the reference to InstantiatedEntries is cleared in dev's code? 

It would also complicate the code by needing a scene reference..

Personally, i prefer dispose.